zhaquirks/tuya/ts011f_plug.py

```diff
@@ -1404,3 +1404,78 @@ class Plug_2AC_var05(EnchantedDevice):
             },
         },
     }
+
+
+class Plug_TZ3000_2AC(CustomDevice):
+    """TS0011F 2 outlet plug."""
+
+    quirk_id = TUYA_PLUG_ONOFF
+
+    signature = {
+        MODEL: "TS011F",
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=81
+            # device_version=1
+            # input_clusters=["0x0000", "0x0004", "0x0005", "0x0006", "0x0702", "0x0b04"]
+            # output_clusters=["0x000a", "0x0019"]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.SMART_PLUG,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    Metering.cluster_id,
+                    ElectricalMeasurement.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id],
+            },
+            # <SimpleDescriptor endpoint=2 profile=260 device_type=81
+            # device_version=1
+            # input_clusters=["0x0004", "0x0005", "0x0006", "0x0702", "0x0b04"]
+            # output_clusters=[25, 10]>
```

Comment:
```suggestion
            # output_clusters=[]>
```

Reply:
fair enough, I don't know enough about this config, so just kept what was suggested to me in the issue, but agree the comments do feel a little redundant.